portafly/src/tests/pages/Accounts.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+import { render } from 'tests/custom-render'
+import { Accounts } from 'pages'
+import { useGetDeveloperAccounts } from 'dal/accounts'
+import accounts from 'tests/examples/accounts.json'
```

Comment:
So instead of importing that example you'd build the necessary data like https://github.com/3scale/porta/pull/1806/files#diff-9f4f2ee10599b0500704e7b16b78965bR5-R9

Reply:
There would still be repetition, right? For this particular case we could use the factory in `tests/examples/accounts` and import them in `Accounts.test` and `AccountsTable.test`.
WDYT?

Reply:
See my [latest commit](https://github.com/3scale/porta/pull/1785/commits/3096438f638b963c297f981443d7d8609266084a).

Reply:
The idea would be remove the "tests/examples/*.json" favoring the factories

Reply:
Exactly. What I mean is to call the factories in `tests/examples/*` so that we don't repeat ourselves.